UPGRADING.md

```diff
@@ -227,14 +227,6 @@ definitions:
 
 - `consensus.skip_timeout_commit` has been removed in favor of `consensus.timeout_commit=0s`.
 
-### Database
-
-- The default database has been changed from `goleveldb` to
```

Comment:
If golevelDB still the default?

Reply:
No. You think we should still mention this in UPGRADING?